### DISCUSSION TITLE
Qual CMS utilizar?

### DISCUSSION THREAD

**Discussion by thiamsantos** [frontendbr/forum#228] [Perguntas e Respostas]
Eu estou começando, e sempre me vêm a mente, quando eu fizer um site para um cliente totalmente leigo que CMS utilizar para construir o site. Todos com quem eu conversei então indicam o WordPress, o problema é que eu não gosto do WordPress e também não preciso de todo o ambiente dele.  Eu gostaria de algo semelhante à uma Template engine, onde eu colocaria as variáveis, e o usuário pudesse customizer o conteúdo delas direto de um painel administrativo simples. E que esse painel administrativo fosse facilmente customizavel. 

Eu procuro algo entre um gerador de site estático e uma CMS, e que fosse preferencialmente escrito em JavaScript. 

Existe alguma ferramenta que encaixe nessas características?

**Comment by barbier** [comment]
Já viu o [Ghost](https://ghost.org/)?

**Comment by ortizvinicius** [comment]
Pra que tipo de conteúdo?

**Comment by thiamsantos** [comment]
Algo mais próximo de site institucional.

**Comment by adeonir** [comment]
Um que achei bem interessante, mas ainda não usei, é o [Grav](https://getgrav.org/).

**Comment by rubenmarcus** [comment]
Esse grav parece ser bem legal!! Ghost e afins em Node, o server em geral nao é mais caro que servers PHP? Pra um site de conteudo geral como esses que encontramos, blogs tipo Nao Salvo,Jovem Nerd, etc etc , wordpress ainda é o mais recomendado? Eu to pensando em fazer um blog sobre arte e cultura, e até hoje só mexi com wordpress,joomla,php nuke e drupal

**Comment by fabricionaweb** [comment]
Rapaz, CMS em PHP é o que nao falta kkkk A dificuldade é escolher mesmo

Um outro que eu posso recomendar é o [Anchor](https://github.com/anchorcms/anchor-cms)
